dev.geopjr.Hashbrown.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: dev.geopjr.Hashbrown
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+sdk: org.gnome.Sdk
+command: hashbrown
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/doc
+  - /share/man
+  - "*.a"
+  - "*.la"
+
+modules:
+  - name: livevent
+    sources:
+      - type: git
+        url: https://github.com/libevent/libevent.git
+        tag: release-2.1.12-stable
+
+  - name: hashbrown
+    buildsystem: simple
+    build-commands:
+      - $(pwd)/crystal/bin/crystal build ./src/hashbrown.cr --no-debug --release
+      - install -D -m 0755 hashbrown /app/bin/hashbrown
+      - install -D -m 0644 extra/Hashbrown.desktop /app/share/applications/dev.geopjr.Hashbrown.desktop
+      - install -D -m 0644 extra/icons/logo.svg /app/share/icons/hicolor/scalable/apps/dev.geopjr.Hashbrown.svg
+    post-install:
+      - install -D -m 0644 extra/dev.geopjr.Hashbrown.metainfo.xml /app/share/metainfo/dev.geopjr.Hashbrown.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/GeopJr/Hashbrown.git
+        tag: v1.0.2
+        commit: 09c2998521ed5b14560b12ebb503bc4835dd9009
+      - type: archive
+        dest: crystal/
+        url: https://github.com/crystal-lang/crystal/releases/download/0.35.1/crystal-0.35.1-1-linux-x86_64.tar.gz
+        sha256: 6c3fd36073b32907301b0a9aeafd7c8d3e9b9ba6e424ae91ba0c5106dc23f7f9
+      - type: git
+        url: https://github.com/jhass/crystal-gobject.git
+        branch: main
```

Comment:
this needs to point to a specific git commit/tag for build reproducibility as per https://github.com/flathub/flathub/wiki/App-Requirements#stable-releases-reproducible-builds

Reply:
The latest tag is missing some important commits, so I will only specify the latest commit.